Add qrcode control

PadreSVK opened the pull request:
Add control for rendering qrcode based on Url to html canvas

PadreSVK committed:
init

exyi commented:
Thanks, this looks quite useful. It has some limitations that should be stated in the README:

1. It requires resource `jquery` to be registered.
1. It only works client-side so it may flash in after some time.
1. It requires some JavaScript library, please add a note where does it come from.
1. QR codes have some settings (most importantly level of error correction) that can be set. Since none of them is possible to set, please add a note about the default values.

The contents of a QR code does not have to be a URL, so I'd rename the property `Url` to something like "content"

PadreSVK committed:
Rename Url property to Content

PadreSVK committed:
Update readme.md

PadreSVK commented:
@exyi

> 1. It requires resource `jquery` to be registered.  

added to readme.md
> 2. It only works client-side so it may flash in after some time.

added to readme.md
> 3. It requires some JavaScript library, please add a note where does it come from.  

added to readme.md
> 4. QR codes have some settings (most importantly level of error correction) that can be set. Since none of them is possible to set, please add a note about the default values.

added to readme.md
> The contents of a QR code does not have to be a URL, so I'd rename the property `Url` to something like "content"

renamed to ```Content```, for ko bindings ```content```
